tests/ha/wait_others_upgraded.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Wait for all nodes which are allowed to upgrade after.
+# Maintainer: Christian Lanig <clanig@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use hacluster;
+use lockapi;
+
+sub run {
+
```

Comment:
I know this is a bit of a nitpick, but the empty line here is not very perl-like. :P

Reply:
yeah I don't know why I often keep an empty line there... thx